### PR TITLE
Handle default package

### PR DESCRIPTION
**Pull request by aalmiray**
Fix for #4

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-    }
+plugins {
+    id 'groovy'
+    id 'idea'
+    id 'maven-publish'
+    id 'java-gradle-plugin'
+    id 'com.jfrog.bintray'              version '1.8.4'
+    id 'net.nemerosa.versioning'        version '2.7.1'
+    id 'com.github.ben-manes.versions'  version '0.20.0'
+    id 'com.gradle.plugin-publish'      version '0.10.0'
```
Comment (bmuschko) [inline build.gradle]: For this plugin we won't need the portal publish plugin. It's already synced from JCenter to the portal. Can you remove this plugin and its configuration below?

File: build.gradle [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-    dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:0.6'
-    }
+plugins {
+    id 'groovy'
+    id 'idea'
+    id 'maven-publish'
+    id 'java-gradle-plugin'
+    id 'com.jfrog.bintray'              version '1.8.4'
+    id 'net.nemerosa.versioning'        version '2.7.1'
```
Comment (bmuschko) [inline build.gradle]: I personally won't use these two plugins. Let's remove them for now.

File: build.gradle [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:0.6'
-    }
+plugins {
+    id 'groovy'
+    id 'idea'
+    id 'maven-publish'
```
Comment (bmuschko) [inline build.gradle]: No need for this one I believe. The Bintray plugin already applies it internally.

File: build.gradle [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:0.6'
-    }
+plugins {
+    id 'groovy'
+    id 'idea'
```
[the remark this answers is not in the document]
Comment (aalmiray) [reply build.gradle]: OK

File: build.gradle [hunk 5]
```diff
@@ -52,7 +53,7 @@ artifacts {
 }
 
 idea.project {
```
[the remark this answers is not in the document]
Comment (aalmiray) [reply build.gradle]: OK

File: build.gradle [hunk 6]
```diff
@@ -131,4 +134,33 @@ bintray {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+gradlePlugin {
```
[the remark this answers is not in the document]
Comment (aalmiray) [reply build.gradle]: Yes. I removed the resource file under `META-INF/gradle-plugins`.

File: build.gradle (hunk 2 above)
Comment (aalmiray) [reply build.gradle]: OK

File: build.gradle (hunk 3 above)
Comment (aalmiray) [reply build.gradle]: OK

File: build.gradle (hunk 1 above)
Comment (aalmiray) [reply build.gradle]: Hmm OK, though I think this was the preferred way to publish plugins

Comment (bmuschko) [reply build.gradle]: It is the preferred way but it's already set up to sync automatically. Not sure how the portal would behave if you'd actually publish there and sync.

**Pull request merged**